src/App.js

```diff
@@ -14,11 +15,84 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    isLoad: false,
+    goods: [...goodsFromServer],
+    goodsSelected: [...goodsFromServer],
```

Comment:
почему селектед? это нетронутый массив
```suggestion
    originalGoods: [...goodsFromServer],
```